CHANGELOG.md

```diff
@@ -1,5 +1,51 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2025-02-20
+
+Monitor the release status by region at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled `v20250220`.
+
+### Announcements
+* HTTP Application Routing (preview) is going to be [retired on March 3, 2025](https://azure.microsoft.com/updates?id=retirement-http-application-routing-addon-preview-for-aks-will-retire-03032025) and AKS will start to block new cluster creation with HTTP Application Routing (preview) enabled. Affected clusters must migrate to the generally available Application Routing add-on prior to that date. Refer to the [migration guide](https://learn.microsoft.com/en-us/azure/aks/app-routing-migration) for more information.
+
+### Release Notes
+* Features:
+   * Application routing add-on support for [configuring the default NGINX ingress controller visibility](https://learn.microsoft.com/azure/aks/app-routing-nginx-configuration?tabs=azurecli#control-the-default-ingress-controller-configuration-when-creating-the-cluster) is now generally available in API 2025-02-01.
+
+   * AKS Kubernetes version 1.32 is now being rolled out and is expected to reach all regions on or before the end of March. Please use the [az-aks-get-versions](https://learn.microsoft.com/azure/aks?view=azure-cli-latest#az-aks-get-versions) command to accurately capture if Kubernetes version 1.32 is available in your region.
+   * AKS Kubernetes patch versions 1.29.12, 1.29.13, 1.30.8, 1.30.9, 1.31.4, and 1.31.5 are now available.
+   * 
+
+
+* Preview Features:
+   * AKS Kubernetes version 1.32 is now in preview.
+
+
+* Bug Fixes:
+   * Resolved an issue with Istio service mesh add-on where having multiple operations with the Lua EnvoyFilter was not allowed.
+   *  
+
+
+* Behavior Changes:
+   * Starting in the Karpenter v0.6 image, Node Auto Provisioning (NAP) will bootstrap nodes from Scriptless to AgentBaker using custom data provided by Node Provisioning Service (NPS) and improve the stability of NAP.
+
+* Component Updates:
+   * Beginning in Kubernetes 1.32, AKS uses CNS to install Azure CNI NodeSubnet by default, meaning users will now see the CNS daemonset in their environments with no changes to functionality or required action.
+   * ExtensionManager has been upgraded to  [v1.22.4](https://github.com/azure-core/ClusterConfigurationAgent/releases/tag/v1.22.4).
```

Comment:
this repo is private plus extension manager is being moved to AKS service hosted components, we should remove this line